public/wp-content/themes/newsup-child/single-anime.php

```diff
@@ -123,6 +128,71 @@
                         </div>
                     </div>
 
+<!--                    print associated anime reviews in another card-->
+                  <?php
+                  $reviews_query = new WP_Query(array(
+                      'posts_per_page' => -1,
+                      'post_type'=> 'anime_review',
+                      'orderby' => 'title',
+                      'order' => 'ASC',
+                      'meta_query' => array(
+                          array(
+                              'key' => 'review_anime',
+                              'compare' => 'LIKE',
+                              'value' => '"' . get_the_ID() . '"' // ensure we compare strings not ints
+                          )
+                      )
+                  ));
+                  $reviews_array = get_posts(array(
+                      'posts_per_page' => -1,
+                      'post_type'=> 'anime_review',
+                      'orderby' => 'title',
+                      'order' => 'ASC',
+                      'meta_query' => array(
+                          array(
+                              'key' => 'review_anime',
+                              'compare' => 'LIKE',
+                              'value' => '"' . get_the_ID() . '"' // ensure we compare strings not ints
+                          )
+                      )
+                  ));
+                  $reviews_posts = array();
+                  foreach ($reviews_array as $item) {
+                      array_push($reviews_posts, setup_postdata($item));
+                  }
+                  echo (gettype($reviews_query)) . '<br>'; //array of associated review posts
+                  echo (gettype($reviews_array)) . '<br>';
+                  echo (gettype($reviews_posts)) . '<br>';
```

Comment:
This needs to be removed. This echos to the front end.

Reply:
yea mb, that was for testing. i will remove it for sure